src/QuadGK.jl

```diff
@@ -318,10 +318,14 @@ function eigvec1(b,z::Number,m=length(b)+1)
     return v
 end
 
-# return (x, w) pair of N quadrature points x[i] and weights w[i] to
-# integrate functions on the interval (-1, 1).  i.e. dot(f(x), w)
-# approximates the integral.  Uses the method described in Trefethen &
-# Bau, Numerical Linear Algebra, to find the N-point Gaussian quadrature
+"""
+    gauss([T,] N)
+
+Return a pair `(x, w)` of `N` quadrature points `x[i]` and weights `w[i]` to
+integrate functions on the interval `(-1, 1)`,  i.e. `dot(f(x), w)`
```

Comment:
`dot(w, f.(x))`, so that you don't conjugate `f`.  Or `sum(w .* f.(x))` is probably more natural here.